runtime/kusama/src/weights/runtime_common_paras_registrar.rs

```diff
@@ -42,28 +43,64 @@ use sp_std::marker::PhantomData;
 /// Weight functions for `runtime_common::paras_registrar`.
 pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> runtime_common::paras_registrar::WeightInfo for WeightInfo<T> {
+	// Storage: Registrar NextFreeParaId (r:1 w:1)
+	// Storage: Registrar Paras (r:1 w:1)
+	// Storage: Paras ParaLifecycles (r:1 w:0)
 	fn reserve() -> Weight {
-		(48_931_000 as Weight)
+		(23_240_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(3 as Weight))
 			.saturating_add(T::DbWeight::get().writes(2 as Weight))
 	}
+	// Storage: Registrar Paras (r:1 w:1)
+	// Storage: Paras ParaLifecycles (r:1 w:1)
+	// Storage: Configuration ActiveConfig (r:1 w:0)
+	// Storage: Paras PvfActiveVoteMap (r:1 w:0)
+	// Storage: Paras CodeByHash (r:1 w:1)
+	// Storage: ParasShared CurrentSessionIndex (r:1 w:0)
+	// Storage: Paras ActionsQueue (r:1 w:1)
+	// Storage: Paras CodeByHashRefs (r:1 w:1)
+	// Storage: Paras CurrentCodeHash (r:0 w:1)
+	// Storage: Paras UpcomingParasGenesis (r:0 w:1)
 	fn register() -> Weight {
-		(4_193_117_000 as Weight)
-			.saturating_add(T::DbWeight::get().reads(5 as Weight))
-			.saturating_add(T::DbWeight::get().writes(4 as Weight))
+		(8_646_169_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(8 as Weight))
```

Comment:
The number of Read/Writes should only change when there was a logic change that did not update the weights.  
Happened in Substrate quite often but I don't know about Polkadot.